frontend/js/bulkSearch.js

```diff
@@ -441,3 +441,40 @@ $(document)
         .then(result => drawPaginationComponent(1, result.count))
         .then(() => fetchMeltingCurves(Array.from(selectedExperiments), Array.from(selectedProteins)))
     );
+
+
+
+
+TourHelper.attachTour('#help-menu-item', [
+    {
+        target: '#spacedText',
+        content: 'This page is an advanced search page.',
+        placement: ['bottom']
+    },
+    {
+        target: '#experiment-list',
+        content: 'First select which experiments you want to inspect further.',
+        placement: ['bottom']
+    },
+    {
+        target: '#protein-input',
+        content: 'Then enter UniProdId\'s which interest you.',
+        placement: ['bottom']
+    },
+    {
```

Comment:
Yes, I thought about that too, the issues are: I have to presume the database is populated with the data that I chose. So as long as we leave the experiments of Chris in there, visible, it would work.
For now, I'm doing that, then the help looks really nice with data.